### DISCUSSION TITLE
The tutorial page doesn't look right

### DISCUSSION THREAD

**Discussion by xulunfan** [awslabs/gluonts#2115] [Q&A] [answered]
The tutorial page doesn't look right：
<img width="1397" alt="image" src="https://user-images.githubusercontent.com/5166598/176585811-d4e78a7b-dae4-4df2-9cac-f1e46e48ad18.png">

https://ts.gluon.ai/stable/tutorials/forecasting/quick_start_tutorial.html

**Comment by jaheba** [comment, marked as answer]
The page should now be fixed, and have a new design :)
